content/applications/inventory_and_mrp/inventory/management/reporting/inventory_valuation_config.rst

```diff
@@ -32,85 +31,167 @@ between locations in a company's inventory.
 .. _inventory/inventory_valuation_config/accounting:
 
 Types of accounting
--------------------
+===================
 
 Accounting entries will depend on the accounting mode: *Continental* or *Anglo-Saxon*.
 
 .. tip::
    Verify the accounting mode by activating the :ref:`developer-mode` and navigating to
-   :menuselection:`Accounting --> Configuration --> Settings`.
+   :menuselection:`Accounting app --> Configuration --> Settings`.
+
+   Then, in the search bar, look for :guilabel:`Anglo-Saxon Accounting`, to see if the feature is
+   enabled. If it is **not** enabled, Continental accounting mode is in use.
+
+   .. image:: inventory_valuation_config/anglo-saxon.png
+      :align: center
+      :alt: Show the Anglo-Saxon accounting mode feature.
 
 In *Anglo-Saxon* accounting, the costs of goods sold (COGS) are reported when products are sold or
 delivered. This means that the cost of a good is only recorded as an expense when a customer is
-invoiced for a product. *Interim Stock Accounts* are used for the input and output accounts, and are
-both *Asset Accounts* in the balance sheet.
+invoiced for a product. So, set the *Expense* account type to *Cost of Revenue* or *Expenses*.
 
 In *Continental* accounting, the cost of a good is reported as soon as a product is received into
-stock. Additionally, a single *Expense* account is used for both input and output accounts in
-the balance sheet.
+stock.
```

Comment:
If I understand PHT's feedback correctly (please confirm with an expert):
```suggestion
invoiced for a product. So for for **manual** valuation method set the :guilabel:`Expense Account`
to `Stock` for the current asset type; for **automatic** valuation method, set the the
:guilabel:`Expense Account` to an *Expenses* or a *Cost of Revenue* type (e.g. `Cost of Production`,
`Cost of Goods Sold`, etc.).

In *Continental* accounting, the cost of a good is reported as soon as a product is received into
stock. Because of this, the :guilabel:`Expense Account` can be set to **either** *Expenses* or a
*Cost of Revenue* type, however, it is more commonly set to an *Expenses* account.
```

Reply:
You can optionally add an internal ref link to line 117, where the table covers everything in detail